opcua/server/internal_subscription.py

```diff
@@ -140,10 +142,12 @@ def _delete_monitored_items(self, mid):
 
     def datachange_callback(self, handle, value, error=None):
         if error:
-            self.logger.info("subscription %s: datachange callback called with handle '%s' and erorr '%s'", self, handle, error)
+            self.logger.info("subscription %s: datachange callback called with handle '%s' and erorr '%s'",
```

Comment:
>> If you use PyCharm 

after soon 20 years vim, my mind is hardcoded for it. But vim also checks pep8, syntax, etc so it is just a question on what is most readable...